tests/test_web_exceptions.py

```diff
@@ -232,3 +232,16 @@ async def show(request):
     assert str(resp.url)[-5:] == "/show"
     text = await resp.text()
     assert text == "works"
+
+
+@pytest.mark.xfail(
+    raises=AttributeError,
+    reason="Regression in v3.7: https://github.com/aio-libs/aiohttp/issues/5657",
+)
+def test_body_unauthorized() -> None:
+    with pytest.warns(DeprecationWarning):
+        resp = web.HTTPUnauthorized(
+            headers={"X-Custom": "value"}, content_type="custom", body="text"
```

Comment:
Are the headers and content type necessary to reproduce this issue? If not, drop them to avoid confusion.

Reply:
dropped